static/src/javascripts/lib/config.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import pad from 'lib/pad';
```

Comment:
Yeah, just read that :/ I guess we need to be clear about what is polyfilled and what isn't - ideally everything (maybe instance methods get polyfilled in the global scope everything else with this)